source/_addonStore/network.py

```diff
@@ -49,7 +50,11 @@ def _getCurrentApiVersionForURL() -> str:
 
 def _getAddonStoreURL(channel: Channel, lang: str, nvdaApiVersion: str) -> str:
 	_baseURL = "https://nvaccess.org/addonStore/"

```

Comment:
this should probably be removed as it is unused

Reply:
In case it was confusing the suggestion here as to remove the unused variable `_baseURL`